.github/workflows/ci.yml

```diff
@@ -5,6 +5,11 @@ on: [push, pull_request]
 env:
   DEFAULT_JDK_VERSION: 11
 
+concurrency:
```

Comment:
This will stop old CIs to run even if there are new commits (and save space etc.).

cc. @stianst @abstractj